validated/dash_apps/finished_apps/daily_validation.py

```diff
@@ -1,15 +1,17 @@
 from datetime import datetime
 from decimal import Decimal
 
-import plotly.express as px
+import plotly.graph_objects as go
 from dash import dcc, html
 from django_plotly_dash import DjangoDash
 
 from station.models import Station
-from validated.functions import daily_validation
+from validated.functions import daily_validation, detail_list
+from validated.tables import create_daily_table, create_detail_table
 from variable.models import Variable
 
-# Create a Dash app
+DEFAULT_FONT = "Open Sans, Raleway, Dosis, Ubuntu, sans-serif"
+
 app = DjangoDash("DailyValidation")
 
 # Filters (in final app this will get data from forms)
```

Comment:
I'm not logged in and all this DB querying works fine. Nothing to worry about at this time, as this part is under development, but at some point this will need to go thorugh the authentication system - specially the validation view, not so much @CWestICL 's reporting view.

Reply:
I'll add this as a separate issue